recipes-kernel/linux/files/5.x-kernel/0002-Workaround-to-correct-DP-clock-to-154MHZ.patch

```diff
@@ -0,0 +1,35 @@
+From a76d54e938efd65ac60c5b90ee57b4a118bba102 Mon Sep 17 00:00:00 2001
+From: Chao Zeng <chao.zeng@siemens.com>
+Date: Wed, 6 Jan 2021 15:51:18 +0800
+Subject: [PATCH 02/14] Workaround to correct DP clock to 154MHZ.
+
+BUG: as the default DPI source should be DPI_1_IN_CLK
+     but seems the DSS input clk swap the DPI0 and DPI1
+     so change the DPI clock to DPI0 to provide the clock
+     to DPI
+
```

Comment:
This patch is still needed (recently validated)? The description is a bit hard to read for me.

Which boards are affected? And/or which SOC revisions? Does this also affect the EVMs of TI? When does this trigger / how to reproduce?

Reply:
For PG2 boards using the new sysfw. using the DPI clock source can not get the correct DP clock. 
should change to the other DP source to get the correct DP clock.

I have verified the DP function on the TI EVM,there is no problem.

I have forward the e-mail that I discuss with TI engineer.